ngraph_bridge/enable_variable_ops/ngraph_replace_variable_modifiers.cc

```diff
@@ -35,6 +35,29 @@ namespace ngraph_bridge {
 // 1. ReplaceModifiers
 // 2. RewriteForTracking
 
+Status CreateBinaryOpNode(NodeBuilder::NodeOut* ndef_gen_op,
+                          std::string new_name_op, std::string op_type,
+                          NodeBuilder::NodeOut input1, NodeBuilder::NodeOut input2, DataType dtype, Node*& node,
+                          Graph*& graph, bool replace_input)
+                           {
+  Node* generic_op;
+  new_name_op = node->name() + new_name_op;
```

Comment:
Here is a sample call to this function 
```
string new_name_mul = node->name() + "_Mul";
      NodeBuilder::NodeOut ndef_mul_op;
      TF_RETURN_IF_ERROR(CreateBinaryOpNode(&ndef_mul_op, new_name_mul, "Mul",
                                            input_accum, input_momentum, dtype,
                                            node, graph, true));
```

I think `node->name()` is being added twice.